src/data/entity_registry.ts

```diff
@@ -66,6 +66,12 @@ const subscribeEntityRegistryUpdates = (conn, store) =>
     "entity_registry_updated"
   );
 
+export const subscribeEntityRegistryEntryUpdated = (conn, callback) =>
+  conn.subscribeEvents(
+    debounce((event) => callback(event), 500, true),
```

Comment:
The callback should debounce itself, we should not make that decision in the subscribe. Also, why not just subscribe to the entity registry itself ?

Reply:
the ER always returns everything no? I need just the individual update event that has the old and new id's on it.... or are you saying use subscribe message and catch just that event rather than exposing this here?

Reply:
Or did you mean to do this?